### PR TITLE
Add terminal newline

### PR DESCRIPTION
**Pull request by adamtherookie**

### CODE REVIEW

**Review by Ayman0x03 — COMMENTED** [review]
Did you test it ?

File: kernel.c [hunk 1]
```diff
@@ -88,8 +88,9 @@ void terminal_putchar(char c)
 	terminal_putentryat(c, terminal_color, terminal_column, terminal_row);
 	if (++terminal_column == VGA_WIDTH) {
 		terminal_column = 0;
-		if (++terminal_row == VGA_HEIGHT)
-			terminal_row = 0;
+		if (c == "\n")
+			terminal_row ++;
+            terminal_column = 0;
```
Comment (adamtherookie) [inline kernel.c:93]: This needs to be indented. Don't know why it isn't

**Comment by adamtherookie** [comment]
Yeah, on my machine, but now one line needs to be indented. Don't know why it isnt

**Comment by Ayman0x03** [comment]
Not Built Build it and test it

**Comment by adamtherookie** [comment]
Just a sec

**Pull request closed without merging**